CMakeLists.txt

```diff
@@ -210,6 +217,7 @@ if(NOT WIN32)
         core_optional_dep(LibDRM ENABLE_MIR)
       else()
         core_optional_dep(X ENABLE_X11)
+        core_optional_dep(XRandR ENABLE_X11)
```

Comment:
for X11 builds the module FindROCKCHIP should not be needed. X11 is either build with OpenGL or GLES but not for a specific hardware vendor.